src/shared/components/ui/List/List.module.scss

```diff
@@ -230,10 +229,10 @@
 
     li:before {
       font-family: 'Font Awesome 5 Free';
-			content: '\f00c';
+      content: '\f00c';
       font-size: 14px;
-			font-weight: 600;
-			color: $color-brand;
+      font-weight: 600;
+      color: $color-brand;
       position: absolute;
       left: 0;
```

Comment:
Musiałbyś użyć nowego fonta bo nie ma już fontawesome

Reply:
@fifijo hmm jakaś sugestia na jaki font to zamienić ? :)

Reply:
@OskarJankowiak-Landingi na ten co używamy w Icon. Sprawdź komponent Icon i dodatkowo tutaj https://landingi.atlassian.net/wiki/spaces/PROJ/pages/279642546/Podmiana+ikon#Pliki pobierz plik fontelo. Folder zawiera plik demo z wszystkimi ikonkami i kodami, które możesz użyć w content: ''